src/platform/packages/shared/kbn-standalone-packaging-utils/src/mocked_services/injected_metadata.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { InternalInjectedMetadataSetup } from '@kbn/core-injected-metadata-browser-internal';
+
+export interface InjectedMetadataOptions {
+  kibanaBranch?: string;
+  kibanaVersion?: string;
+  kibanaBuildNumber?: number;
+  basePath?: string;
+  serverBasePath?: string;
+  publicBaseUrl?: string;
+  elasticsearchClusterUuid?: string;
+  elasticsearchClusterName?: string;
+  elasticsearchClusterVersion?: string;
+  themeDarkMode?: 'light' | 'dark';
+  themeName?: string;
+}
+
+/**
+ * Creates a mocked InternalInjectedMetadataSetup for standalone packaging.
+ * This allows Kibana plugins to run without the full Kibana server context.
+ *
+ * @param options - Configuration options to customize the mocked metadata
+ * @returns A mock InternalInjectedMetadataSetup implementation
+ *
+ * @example
+ * ```ts
+ * const metadata = createMockedInjectedMetadata({
+ *   kibanaVersion: '8.12.0',
+ *   kibanaBuildNumber: 54321,
+ *   basePath: '/my-app',
+ * });
+ * ```
+ */
+export function createMockedInjectedMetadata(
+  options: InjectedMetadataOptions = {}
+): InternalInjectedMetadataSetup {
+  const {
+    kibanaBranch = 'main',
+    kibanaVersion = '9.1.0',
+    kibanaBuildNumber = 12345,
+    basePath = '',
+    serverBasePath = '',
+    publicBaseUrl = '',
+    elasticsearchClusterUuid = 'test-cluster-uuid',
+    elasticsearchClusterName = 'test-cluster-name',
+    elasticsearchClusterVersion = '8.0.0',
+    themeDarkMode = 'light',
+    themeName = 'default',
+  } = options;
```

Comment:
These don't seem to be used AFAICS, but are required to be valid in order for all the services to work